### PR TITLE
fix(docs): configuration.copy link

### LINKED ISSUE #708: Incorrect documentation link for `configuration.copy` in table
In `table.ts` link for `configuration.copy` has been mention as [http://goo.gl/dKWIyS](http://goo.gl/dKWIyS) that is redirect to incorrect page now.

### PR DESCRIPTION
**Pull request by laljikanjareeya**
Fixes #708

### CODE REVIEW

File: src/table.ts [hunk 1]
```diff
@@ -584,9 +584,8 @@ class Table extends common.ServiceObject {
    * table.copy(yourTable, (err, apiResponse) => {});
    *
    * //-
-   * // See the <a href="http://goo.gl/dKWIyS">`configuration.copy`</a> object
-   * for
-   * // all available options.
+   * // See the https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy
```
Comment (stephenplusplus) [inline src/table.ts:587]:
```suggestion
   * // See https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy
```

File: src/table.ts [hunk 2]
```diff
@@ -644,8 +643,8 @@ class Table extends common.ServiceObject {
    *     source table(s) to copy data from.
    * @param {object=} metadata Metadata to set with the copy operation. The
    *     metadata object should be in the format of the
```
Comment (stephenplusplus) [inline src/table.ts:645]:
```suggestion
   *     metadata object should be in the format of a
```

File: src/table.ts [hunk 3]
```diff
@@ -644,8 +643,8 @@ class Table extends common.ServiceObject {
...
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     [`configuration.copy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
```
Comment (stephenplusplus) [inline src/table.ts:646]:
```suggestion
   *     [`JobConfigurationTableCopy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
```

File: src/table.ts [hunk 4]
```diff
@@ -644,8 +643,8 @@ class Table extends common.ServiceObject {
...
+   * property of a Jobs resource.
```
Comment (stephenplusplus) [inline src/table.ts:647]:
```suggestion
   * object.
```

File: src/table.ts [hunk 5]
```diff
@@ -670,9 +669,8 @@ class Table extends common.ServiceObject {
    * table.copyFrom(sourceTables, (err, apiResponse) => {});
    *
    * //-
-   * // See the <a href="http://goo.gl/dKWIyS">`configuration.copy`</a> object
-   * for
-   * // all available options.
+   * // See the https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy
```
Comment (stephenplusplus) [inline src/table.ts:672]:
```suggestion
   * // See https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy
```

File: src/table.ts [hunk 6]
```diff
@@ -751,9 +749,8 @@ class Table extends common.ServiceObject {
    * });
    *
    * //-
-   * // See the <a href="http://goo.gl/dKWIyS">`configuration.copy`</a> object
-   * for
-   * // all available options.
+   * // See the https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy
```
Comment (stephenplusplus) [inline src/table.ts:752]:
```suggestion
   * // See https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy
```

File: src/table.ts [hunk 7]
```diff
@@ -870,9 +867,8 @@ class Table extends common.ServiceObject {
    * table.createCopyFromJob(sourceTables, callback);
    *
    * //-
-   * // See the <a href="http://goo.gl/dKWIyS">`configuration.copy`</a> object
-   * for
-   * // all available options.
+   * // See the https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy
```
Comment (stephenplusplus) [inline src/table.ts:870]:
```suggestion
   * // See https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy
```

File: src/table.ts [hunk 8]
```diff
@@ -839,8 +836,8 @@ class Table extends common.ServiceObject {
    *     source table(s) to copy data from.
    * @param {object} [metadata] Metadata to set with the copy operation. The
    *     metadata object should be in the format of the
```
Comment (stephenplusplus) [inline src/table.ts:838]:
```suggestion
   *     metadata object should be in the format of a
```

File: src/table.ts [hunk 9]
```diff
@@ -839,8 +836,8 @@ class Table extends common.ServiceObject {
...
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     [`configuration.copy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
```
Comment (stephenplusplus) [inline src/table.ts:839]:
```suggestion
   *     [`JobConfigurationTableCopy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
```

File: src/table.ts [hunk 10]
```diff
@@ -839,8 +836,8 @@ class Table extends common.ServiceObject {
...
+   * property of a Jobs resource.
```
Comment (stephenplusplus) [inline src/table.ts:840]:
```suggestion
   * object.
```

File: src/table.ts [hunk 11]
```diff
@@ -726,8 +724,8 @@ class Table extends common.ServiceObject {
    * @param {Table} destination The destination table.
    * @param {object} [metadata] Metadata to set with the copy operation. The
    *     metadata object should be in the format of the
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     [`configuration.copy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
+   * property of a Jobs resource.
```
Comment (stephenplusplus) [inline src/table.ts:728]:
```suggestion
   * object.
```

File: src/table.ts [hunk 12]
```diff
@@ -726,8 +724,8 @@ class Table extends common.ServiceObject {
    * @param {Table} destination The destination table.
    * @param {object} [metadata] Metadata to set with the copy operation. The
    *     metadata object should be in the format of the
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     [`configuration.copy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
```
Comment (stephenplusplus) [inline src/table.ts:727]:
```suggestion
   *     [`JobConfigurationTableCopy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
```

File: src/table.ts [hunk 13]
```diff
@@ -726,8 +724,8 @@ class Table extends common.ServiceObject {
    * @param {Table} destination The destination table.
    * @param {object} [metadata] Metadata to set with the copy operation. The
    *     metadata object should be in the format of the
```
Comment (stephenplusplus) [inline src/table.ts:726]:
```suggestion
   *     metadata object should be in the format of a
```

File: src/table.ts [hunk 14]
```diff
@@ -561,8 +561,8 @@ class Table extends common.ServiceObject {
...
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     [`configuration.copy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
+   * property of a Jobs resource.
```
Comment (stephenplusplus) [inline src/table.ts:565]:
```suggestion
   * object.
```

File: src/table.ts [hunk 15]
```diff
@@ -561,8 +561,8 @@ class Table extends common.ServiceObject {
    * @param {Table} destination The destination table.
    * @param {object} [metadata] Metadata to set with the copy operation. The
    *     metadata object should be in the format of the
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     [`configuration.copy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
```
Comment (stephenplusplus) [inline src/table.ts:564]:
```suggestion
   *     [`JobConfigurationTableCopy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
```

File: src/table.ts [hunk 16]
```diff
@@ -561,8 +561,8 @@ class Table extends common.ServiceObject {
    * @param {Table} destination The destination table.
    * @param {object} [metadata] Metadata to set with the copy operation. The
    *     metadata object should be in the format of the
```
Comment (stephenplusplus) [inline src/table.ts:563]:
```suggestion
   *     metadata object should be in the format of a
```

**Comment by stephenplusplus** [comment]
@laljikanjareeya sorry that my review makes it look like everything is wrong! I just had some ideas for new phrasing, and tried to leave comments with code suggestions so you could just click to accept them and have the code update automatically.

**Comment by laljikanjareeya** [comment]
@stephenplusplus Thanks for the review, I made changes in one commit manually. Please take a look.

File: src/table.ts [hunk 17]
```diff
@@ -560,9 +560,9 @@ class Table extends common.ServiceObject {
    *
    * @param {Table} destination The destination table.
    * @param {object} [metadata] Metadata to set with the copy operation. The
-   *     metadata object should be in the format of the
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     metadata object should be in the format of a
+   *     [`JobConfigurationTableCopy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
+   * object.
```
Comment (stephenplusplus) [inline src/table.ts:565]:
```suggestion
   * object.
   *     object.
```

File: src/table.ts [hunk 18]
```diff
@@ -643,9 +642,9 @@ class Table extends common.ServiceObject {
    * @param {Table|Table[]} sourceTables The
    *     source table(s) to copy data from.
    * @param {object=} metadata Metadata to set with the copy operation. The
-   *     metadata object should be in the format of the
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     metadata object should be in the format of a
+   *     [`JobConfigurationTableCopy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
+   * object.
```
Comment (stephenplusplus) [inline src/table.ts:647]:
```suggestion
   *     object.
```

File: src/table.ts [hunk 19]
```diff
@@ -725,9 +723,9 @@ class Table extends common.ServiceObject {
    *
    * @param {Table} destination The destination table.
    * @param {object} [metadata] Metadata to set with the copy operation. The
-   *     metadata object should be in the format of the
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     metadata object should be in the format of a
+   *     [`JobConfigurationTableCopy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
+   * object.
```
Comment (stephenplusplus) [inline src/table.ts:728]:
```suggestion
   *     object.
```

File: src/table.ts [hunk 20]
```diff
@@ -838,9 +835,9 @@ class Table extends common.ServiceObject {
    * @param {Table|Table[]} sourceTables The
    *     source table(s) to copy data from.
    * @param {object} [metadata] Metadata to set with the copy operation. The
-   *     metadata object should be in the format of the
-   *     [`configuration.copy`](http://goo.gl/dKWIyS) property of a Jobs
-   * resource.
+   *     metadata object should be in the format of a
+   *     [`JobConfigurationTableCopy`](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationTableCopy)
+   * object.
```
Comment (stephenplusplus) [inline src/table.ts:840]:
```suggestion
   *     object.
```

**Comment by stephenplusplus** [comment]
@laljikanjareeya thank you! I approved, although there's a few quick formatting things.

**Pull request merged**